app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -16,23 +16,51 @@ def passthru
 
   def omniauth_callback(name)
     auth = request.env['omniauth.auth']
+    strategy = request.env['omniauth.strategy']
+    required_org = strategy.options[:required_org]
     provider = UserProvider.find_by(name: auth.provider, uid: auth.uid)
 
+    if auth.provider == 'github' && defined?(required_org) && required_org.present?
```

Comment:
```suggestion
    if auth.provider == 'github' && required_org.present?
```

Method `.present?` could be used for any type of variable. reference: https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html